cmd/bnsd/x/username/handlers.go

```diff
@@ -18,9 +18,9 @@ func RegisterRoutes(r weave.Registry, auth x.Authenticator) {
 	r = migration.SchemaMigratingRegistry("username", r)
 
 	b := NewTokenBucket()
-	r.Handle(RegisterTokenMsg{}.Path(), &registerTokenHandler{auth: auth, bucket: b})
-	r.Handle(TransferTokenMsg{}.Path(), &transferTokenHandler{auth: auth, bucket: b})
-	r.Handle(ChangeTokenTargetsMsg{}.Path(), &changeTokenTargetsHandler{auth: auth, bucket: b})
+	r.Handle(&RegisterTokenMsg{}, &registerTokenHandler{auth: auth, bucket: b})
+	r.Handle(&TransferTokenMsg{}, &transferTokenHandler{auth: auth, bucket: b})
```

Comment:
And this actually enforces `*TransferTokenMsg` is a `weave.Msg`, eliminating need for my home-grown compiler assertions.

Reply:
Yes, this is what I was trying to explain :) Because I do not feel strongly about those type assertions, I have put them all back.